modules/cpmso-feat-failure-recovery.adoc

```diff
@@ -0,0 +1,7 @@
+// Module included in the following assemblies:
+//
+// * machine_management/cpmso-about.adoc
+
+:_content-type: CONCEPT
+[id="cpmso-feat-failure-recovery_{context}"]
+= Recovery of the failed control plane machines
```

Comment:
This section is commented out in the build and must be reviewed in a later PR that finishes this section.